app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -275,42 +277,45 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
     throw HttpStatusException(
         HttpStatus.serviceUnavailable, 'GitHub not responding');
   }
+}
 
-  Future<List<String>> _loadBranchRegExps() async {
-    const String path =
-        '/flutter/cocoon/master/app_dart/dev/branch_regexps.txt';
-    final Uri url = Uri.https('raw.githubusercontent.com', path);
+Future<List<String>> loadBranchRegExps(
+    HttpClientProvider branchHttpClientProvider,
+    Logging log,
+    GitHubBackoffCalculator gitHubBackoffCalculator) async {
+  const String path = '/flutter/cocoon/master/app_dart/dev/branch_regexps.txt';
```

Comment:
Side note, doesn't have to be addressed this PR: Was there a reason the branch regexps weren't just stored in the cocoon config? Seems like it would save us this extra HTTP query.

Reply:
HTTP request is intentional: this enables instant access without app re-deployment when some changes are made.